### DISCUSSION TITLE
有个简单的问题

### DISCUSSION THREAD

**Discussion by hqc8848** [subframe7536/maple-font#298] [Q&A] [answered]
想问一下v7.0-beta31里面的NormalNL、NL、Normal以及hinted和unhinted都是什么意思啊，90多个文件根本看不明白
以及想问下怎么校验sha256值（纯好奇）

**Comment by subframe7536** [comment, marked as answer]
命名：https://github.com/subframe7536/maple-font?tab=readme-ov-file#naming-faq
校验：https://github.com/namazso/OpenHashTab
